### PR TITLE
[USETUP][BOOTDATA] Include optional CJK standard fonts

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
This PR enables the developers to include the CJK standard fonts into ReactOS by adding them in Folder `modules/optional`.
JIRA issue: [CORE-9619](https://jira.reactos.org/browse/CORE-9619)
This feature is for test purpose only. You may not use the fonts illegally.

after

### CODE REVIEW

**Comment by blackcrack** [comment]
Reactos is international, and there are not only Japanese fonts, also Mandarin, Indian and many other font's.. they should also possible to testing , so :+1: (a Reactos-dev should able to playing around and learning with it) , but the end - product should stable stable as a rock and working with all fonts from the world where have a Standard in the respective countries. Therefore should it support.

File: dll/win32/syssetup/wizard.c [hunk 1]
```diff
@@ -2046,6 +2147,11 @@ RegistrationProc(LPVOID Parameter)
     RegistrationData->Registered = 0;
     RegistrationData->DefaultContext = SetupInitDefaultQueueCallback(RegistrationData->hwndDlg);
 
+#ifdef __REACTOS__ /* HACK */
```
Comment (HBelusca) [inline dll/win32/syssetup/wizard.c]: Why is this done only now, and not when other data font is set up (currently done in usetup)?

**Comment by katahiromz** [comment]
after

File: dll/win32/syssetup/wizard.c (hunk 1 above)
Comment (katahiromz) [reply dll/win32/syssetup/wizard.c]: OK.

**Pull request merged**